.github/workflows/nightly.yml

```diff
@@ -99,13 +98,15 @@ jobs:
         id: coverity-checkout
         uses: actions/checkout@v2
         with:
-          submodules: 'true'
+          fetch-depth: 0 # otherwise we will have no info about contributors
       - name: Build
         run: |
+          git -C "$GITHUB_WORKSPACE" submodule sync --recursive
```

Comment:
`--recursive`
It's redundant now, we don't have recursive sumbodules. Please, remove it.